modules/sdk/src/main/scala/org/tessellation/sdk/http/p2p/clients/L0TrustClient.scala

```diff
@@ -0,0 +1,29 @@
+package org.tessellation.sdk.http.p2p.clients
+
+import cats.effect.Async
+
+import org.tessellation.schema.peer.L0Peer
+import org.tessellation.schema.trust.TrustScores
+import org.tessellation.sdk.domain.trust.storage.OrdinalTrustMap
+import org.tessellation.sdk.http.p2p.PeerResponse
+import org.tessellation.security.SecurityProvider
+
+import org.http4s.circe.CirceEntityCodec.circeEntityDecoder
+import org.http4s.client.Client
+
+trait L0TrustClient[F[_]] {
+  def getCurrentTrust(l0Peer: L0Peer): F[TrustScores]
+  def getPreviousTrust(l0Peer: L0Peer): F[OrdinalTrustMap]
+}
+
+object L0TrustClient {
+  def make[F[_]: Async: SecurityProvider](client: Client[F]): L0TrustClient[F] =
+    new L0TrustClient[F] {
+
+      def getCurrentTrust(l0Peer: L0Peer): F[TrustScores] =
+        PeerResponse[F, TrustScores]("trust/current")(client).run(l0Peer)
```

Comment:
as written previously, it should be `PeerResponse[F, TrustScores]` not `F[TrustScores]`. there should be no `.run(l0Peer)` in the p2p client.

the trait should look like:
```scala
trait L0TrustClient[F[_]] {
  def getCurrentTrust: PeerResponse[F, TrustScores]
  def getPreviousTrust: PeerResponse[F, OrdinalTrustMap]
}
```

Reply:
Re-worked `L0TrustClient` as requested.